tutorials/source-modeling/plot_source_alignment.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
 """
+
+.. include:: ../../links.inc
```

Comment:
put this line at the bottom of the file, not the top (putting it here is messing up sphinx-gallery's marking up of the title)